src/components/ICA/ICA.js

```diff
@@ -146,15 +188,28 @@ ICA.propTypes = {
 
   /** Specify whether or not the values should be truncated. */
   truncate: PropTypes.bool,
+
+  /** The size of the ICA
+   * @type string
+   */
```

Comment:
Because the type is generated from the propType, we don't need to add the additional `@type` annotation here.

```suggestion
  /** The size of the ICA. */
```